R/capacity/health-inequalities/england/access-availability/waiting-lists.R

```diff
@@ -47,79 +28,107 @@ diagnostics_sliced <-
 diagnostics_vars <-
   diagnostics_sliced |>
   select(
-    `Trust Code` = `Provider Code`,
-    `Waiting 13+ weeks` = `Number waiting 13+ Weeks`
+    trust_code = `Provider Code`,
+    waiting_over_13_weeks = `Number waiting 13+ Weeks`,
+    total_waiting = `Total Waiting List`
   )
 
-# Filter to only open trusts
-# diagnostic_wait_times <-
-diagnostic_open_trusts <-
-  open_trusts |>
-  left_join(
-    diagnostics_vars,
-    by = "Trust Code"
+# NHS Trust table in geographr package -----
+
+# Load in open trusts table created in trust_types.R
+open_trusts <- read_rds("data/open_trust_types.rds")
+
+# Check those in the waiting times data and not in the trusts data
+raw |>
+  anti_join(open_trusts, by = c("Provider Code" = "trust_code")) |>
+  select(`Regional Team Name`, `Provider Name`) |>
+  print(n = Inf)
+# There are many 243 missing in open trusts but notes say 'Data are shown at provider organisation level, from NHS Trusts, NHS Foundation Trusts and Independent Sector Providers'
+# So from names look to be independent providers (e.g. Nuffield, Spire etc)
+# Note here: https://www.england.nhs.uk/statistics/statistical-work-areas/diagnostics-waiting-times-and-activity/monthly-diagnostics-waiting-times-and-activity/
+
+raw |>
+  anti_join(open_trusts, by = c("Provider Code" = "trust_code")) |>
+  select(`Regional Team Name`, `Provider Name`) |>
+  filter(str_detect(`Provider Name`, "(?i)trust")) |>
+  print(n = Inf)
+# No NHS Foundation Trusts
+
+open_trusts |>
+  anti_join(diagnostics_vars) |>
+  print(n = Inf)
+# 51 missing that are ambulance and community trusts
+# In note (https://www.england.nhs.uk/statistics/wp-content/uploads/sites/2/2013/08/DM01-FAQs-v-3.0.pdf) says
+# 'All trusts that provide any of the diagnostic tests that are on the monthly template should complete a return' - so these trusts may not provide these tests.
+
+# Join trust to LAD lookup --------
+
+lookup_trust_lad <- read_feather("R/capacity/health-inequalities/england/trust_calculations/lookup_trust_lad.feather")
```

Comment:
```suggestion
lookup_trust_lad <- read_rds("R/capacity/health-inequalities/england/trust_calculations/lookup_trust_lad.rds")
```
This needs updating to reflect the latest changes to the `.feather` files.